suites/pacific/cephadm/tier-1_multiple_subnet.yaml

```diff
@@ -0,0 +1,229 @@
+  - test:
+      name: Cephadm Bootstrap
+      desc: cephadm cluster bootstrap with mutiple public network
+      module: test_bootstrap.py
+      polarion-id: CEPH-83575132
+      config:
+        command: bootstrap
+        base_cmd_args:
+          verbose: true
+        args:
+          orphan-initial-daemons: true
+          registry-json: registry.redhat.io
+          custom_image: true
+          mon-ip: node1
+          fsid: f64f341c-655d-11eb-8778-fa163e914bcc
+          config:
+            global:
+              public_network: 10.0.104.0/22,10.0.204.0/22
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Host addition with spec file
+      desc: add hosts using spec file with label
+      module: test_cephadm.py
+      polarion-id: CEPH-83574726
+      config:
+        steps:
+          - config:
+              service: host
+              command: set_address
+              args:
+                node: node1
+          - config:
+              service: host
+              command: label_add
+              args:
+                node: node1
+                labels: apply-all-labels
+          - config:
+              command: apply_spec
+              service: orch
+              specs:
+                - service_type: host
+                  address: true
+                  labels: apply-all-labels
+                  nodes:
+                    - node2
+                    - node3
+                - service_type: host
+                  address: true
+                  labels: apply-all-labels
+                  nodes:
+                    - node4
+                - service_type: host
+                  address: true
+                  labels: apply-all-labels
+                  nodes:
+                    - node5
+      abort-on-fail: true
+  - test:
+      name: Apply Monitor service
+      desc: Apply monitor service using label mon
+      module: test_mon.py
+      polarion-id: CEPH-83573772
+      config:
+        command: apply
+        service: mon
+        base_cmd_args:
+          verbose: true
+        args:
+          placement:
+            label: mon
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Apply Manager service
+      desc: Apply manager service using label mgr
+      module: test_mgr.py
+      polarion-id: CEPH-83573777
+      config:
+        command: apply
+        service: mgr
+        base_cmd_args:
+          verbose: true
+        args:
+          placement:
+            label: mgr
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Apply OSD Service
+      desc: Apply OSD service with all available devices
+      module: test_osd.py
+      polarion-id: CEPH-83573735
+      config:
+        command: apply
+        service: osd
+        base_cmd_args:
+          verbose: true
+        args:
+          all-available-devices: true
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
```

Comment:
do we need thsi test case

Reply:
so far now i think we don't have any test suite which using multiple public network configured cluster,
Hence I though it's good to add other service as well.
In order to check other service deployments with different subnet cluster.